wasmer/wasmerStatic.go

```diff
@@ -0,0 +1,41 @@
+package wasmer
```

Comment:
The functions in `wasmerStatic.go` should be private and only accessible via a `wasmerExecutor` instance.

Reply:
Yes, will do in the end